packages/shared/graphql/types.ts

```diff
@@ -225,6 +225,9 @@ export interface Recording {
   userId?: string;
   userRole?: RecordingRole;
   workspace?: Workspace;
+  testRun?: {
+    id: string;
+  } | null;
```

Comment:
nit: This would be easier to work with as a string id:
```suggestion
  testRunId?: string;
```